### PR TITLE
add placeholder for translation text on translation.html

### PR DESCRIPTION
**Pull request by bosticc**

### CODE REVIEW

File: src/main/webapp/translation.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            <li><a href="candidates.html">Candidate Info</a></li>
+            <li><a href="locations.html">Poll Locations</a></li>
+            <li><a href="translation.html">Translate</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <footer>
+    </footer>
+    <script src="script.js"></script>
```
Comment (jdstriker731) [inline src/main/webapp/translation.html]: Everything looks good, but from what either Cam and Zu have said to me, it's best to put the script tags at the very end of the body. This is so the HTML content of the page can always load first

File: src/main/webapp/translation.html [hunk 2]
```diff
@@ -24,7 +24,6 @@
     </header>
     <footer>
     </footer>
-    <script src="script.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
```
Comment (jdstriker731) [inline src/main/webapp/translation.html]: This script tag should go at the bottom too

Comment (bosticc) [reply src/main/webapp/translation.html]: my b

**Review by jdstriker731 — APPROVED** [review]

**Comment by jdstriker731** [comment]
LGTM!

**Pull request merged**